src/usermod.c

```diff
@@ -424,6 +425,9 @@ static /*@noreturn@*/void usage (int status)
 	(void) fputs (_("  -a, --append                  append the user to the supplemental GROUPS\n"
 	                "                                mentioned by the -G option without removing\n"
 	                "                                the user from other groups\n"), usageout);
+	(void) fputs (_("  -r, --remove                  remove the user from only the supplemental GROUPS\n"
+	                "                                mentioned by the -G option without removing\n"
+	                "                                the user from theother groups\n"), usageout);
```

Comment:
To be consistent with `-a` option you should remove `the`.

Reply:
From my point of view no, it's not necessary, but the last word is up to the maintainers.

That reminds me that I haven't tested this PR. I'll try to do that today.

Reply:
> Hi @ikerexxe I fixed the grammatical error.
> 
> In regards to writing tests. Would need to look at how the tests are setup. Never looked at it before so. might have a few questions. Are the tests cases a hard requirement for this PR?

I do worry that if you don't write them now, they'll never get written :)  So if you have time in the next week, that would be great.  They'd go under tests/usertools/, one test per directory.

Unfortunately I haven't yet ported the github repo from travis to github actions, so currently the tests aren't being run.

Reply:
I will try write the tests at a later time and in another commit.